relay-server/src/actors/store.rs

```diff
@@ -1041,6 +1093,7 @@ impl Message for KafkaMessage {
             Self::ReplayEvent(message) => message.replay_id.0,
             Self::ReplayRecordingNotChunked(_message) => Uuid::nil(), // Ensure random partitioning.
             Self::CheckIn(_message) => Uuid::nil(),
+            Self::Span(_) => Uuid::nil(), // TODO
```

Comment:
Q: how does a nil partition key interact with the forwarding?

Reply:
`nil` means random partitioning (see lines above), which should be good for now.